src/app/api/refresh/route.ts

```diff
@@ -0,0 +1,74 @@
+import { NextRequest } from 'next/server'
+
+type Client = {
+	id: string
+	res: any
```

Comment:
Can it be something other than `any`?

Reply:
It's technically this: https://github.com/hashicorp/dev-portal/pull/2615/files#diff-5f0aec67bf02b90279a7a6c4009675ff4574bf755ced6fc905252a7c495e5c70R54.

But I got lazy, especially with just a dev tool.